src/utils/collection-stream.js

```diff
@@ -44,6 +44,12 @@ class WritableCollectionStream extends Writable {
   }
 
   _write(chunk, encoding, next) {
+    if (Array.isArray(chunk)) {
+      const err = new Error('JSON file supplied contains a malformatted document.');
```

Comment:
🤔 I wonder if there's a way we could recover from the error gracefully or maybe we need to change args passing to JSONStream. Can brainstorm more tmrw

Reply:
i slept on it, and actually do feel like this kind of file should be detected as a multiline json file, (it's currently not). I think maybe changing the way that gets parsed will eliminate this error entirely.